src/utils/src/types/project-types.ts

```diff
@@ -42,7 +42,7 @@ export interface Project extends WbsElement {
   goals: DescriptionBullet[];
   features: DescriptionBullet[];
   otherConstraints: DescriptionBullet[];
-  workPackages: WorkPackageSummary[];
+  workPackages: WorkPackage[];
 }
 
 export interface WorkPackageSummary {
```

Comment:
This interface has now been refactored out of service, and can be removed, with it also the workPackageSummaryTransformer can be removed. Make sure to check for any leftover references to things you remove.

Reply:
Sounds good, I'll remove those